File: 프로그래머스/1/12943. 콜라츠 추측/콜라츠 추측.py
def solution(n):
    index = 0
    while True:
        if n == 1:
            return index
        if index >= 500:
            return -1
        if n % 2 == 0:
            n = n // 2
        else:
            n = n * 3 + 1
        index += 1

File: 프로그래머스/1/12943. 콜라츠 추측/콜라츠 추측.swift
func solution(_ num:Int) -> Int {
    var answer : Int = 0
    var n : Int = num
    while (n != 1) {
        if n % 2 == 0 {
            n /= 2
            answer += 1
        }else{
            n *= 3
            n += 1
            answer += 1
        }
    }
    if answer >= 500 {
        answer = -1
    }
    return answer
}